int32 interval
---
bool success
string message
